### PR TITLE
feat: add a new plugin could show system time

### PR DESCRIPTION
**Pull request by Zeroto521**
Plugin demo
demo

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -238,5 +238,16 @@
...
+    },
+    {
+        "ID": "84ddc2a6d43911e8a8d5f2801f1b9fd1",
+        "Name": "Timestamp",
+        "Description": "Show system time and could copy that",
+        "Author": "Zero <Zeroto521>",
+        "Version": "1.0.7",
+        "Language": "python",
+        "Website": "https://github.com/Zeroto521/Flow.Launcher.Plugin.Timestamp",
+        "UrlDownload": "https://github.com/Zeroto521/Flow.Launcher.Plugin.Timestamp/archive/refs/heads/master.zip",
```
Comment (Zeroto521) [inline plugins.json:250]: Note: I don't use github release. That could be a bug to the updater.

Comment (taooceros) [reply plugins.json:250]: I remember I check whether the UrlDownload is from Github Release so it shouldn't be a problem.

Comment (Zeroto521) [reply plugins.json:250]: I have already considered using https://github.com/PyGithub/PyGithub to request GitHub after #50 merged.

Comment (jjw24) [reply plugins.json:250]: > You are right, we should add a fix to that before merging this.

should be fine to merge, @Zeroto521 can change to github release or we use PyGithub.

out of curiosity, @Zeroto521 why arent you using Github Release?

**Review by jjw24 — APPROVED** [review]

File: plugins.json (hunk 1 above)
Comment (Zeroto521) [reply plugins.json:250]: No more special reason, just haven't at present.

**Pull request merged**